test/actions/katello/host/applicable_errata_install_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'katello_test_helper'
+
+module ::Actions::Katello::Host::Erratum
+  class TestBase < ActiveSupport::TestCase
+    include Dynflow::Testing
+    include Support::Actions::Fixtures
+
+    class Installtest < TestBase
+      let(:action_class) { ::Actions::Katello::Host::Erratum::ApplicableErrataInstall }
+
+      let(:uuid) { 'uuid' }
+      let(:content_facet) { mock('a_system').mimic!(::Katello::Host::ContentFacet) }
+      let(:host) do
```

Comment:
I'd like to see one of the host fixtures here  rather than a mock - I think we have some with content facets set up!